### PR TITLE
Merged invoice_list.html

### PR DESCRIPTION
**Pull request by julianandres-eb**
…-list.html merged into templates/invoices_app/invoice-list.html

### CODE REVIEW

File: invoices_app/views.py [hunk 1]
```diff
@@ -53,18 +53,21 @@ def get_queryset(self):
...
 class SupplierInvoiceListView(LoginRequiredMixin, PaginationMixin, ListView):
-    template_name = 'supplier_app/invoice-list.html'
+    template_name = 'invoices_app/invoice-list.html'
     model = Invoice
     paginate_by = 10
 
     def get_queryset(self):
         tax_payer = get_object_or_404(TaxPayer, id=self.kwargs['taxpayer_id'])
         queryset = Invoice.objects.filter(taxpayer=tax_payer.id).order_by('id')
+        import ipdb; ipdb.set_trace()
```
Comment (gonzalogEB) [inline invoices_app/views.py]: Take care here, remove it after debugging

File: invoices_app/views.py [hunk 2]
```diff
@@ -53,18 +53,21 @@ def get_queryset(self):
...
         return queryset
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context['taxpayer_id'] = self.kwargs['taxpayer_id']
+        import ipdb; ipdb.set_trace()
```
Comment (gonzalogEB) [inline invoices_app/views.py]: Also here

File: invoices_app/views.py [hunk 3]
```diff
@@ -53,18 +53,21 @@ def get_queryset(self):
...
+        # context['taxpayer_id'] = self.kwargs['taxpayer_id']
```
Comment (gonzalogEB) [inline invoices_app/views.py]: If it won't be used, remove it

File: invoices_app/views.py (hunk 2 above)
Comment (arielaranda-eb) [inline invoices_app/views.py]: ups!

File: templates/invoices_app/invoice-list.html [hunk 4]
```diff
@@ -28,18 +33,36 @@ <h1 class="text-center"> {% trans "My Invoices"%}</h1>
...
             {% endif %}
             {% for invoice in object_list %}
                 <tr>
                     <th scope="row">{{invoice.id}}</th>
                     <td>{{invoice.invoice_date_received}}</td>
-                    <td><span class="badge badge-success">{{invoice.status}}</span></td>
+                    <td>
+                      <span>
+                        <strong>Status:</strong>
+                        {% if invoice.status == 'NEW' or  invoice.status == 'PAID' or invoice.status == 'APPROVED' %}
```
Comment (gonzalogEB) [inline templates/invoices_app/invoice-list.html]: Here you could use the constants defined for the different status. Check this comment https://github.com/evbeda/suppliers_management/pull/81#discussion_r330220817

**Pull request closed without merging**